src/stravalib/model.py

```diff
@@ -801,8 +794,23 @@ class SegmentEffortAchievement(BaseModel):
     effort_count: Optional[int] = None
 
 
+class SummarySegmentEffort(strava_model.SummarySegmentEffort):
+    # Override superclass fields to match actual Strava API responses
+    activity_id: Optional[int] = Field(
+        validation_alias=AliasChoices("pr_activity_id", "activity_id"),
+        default=None,
+    )
+    elapsed_time: Optional[int] = Field(
+        validation_alias=AliasChoices("pr_elapsed_time", "elapsed_time"),
+        default=None,
+    )
```

Comment:
These are the new aliases that should handle both flavors (i.e., with or without `pr_` prefix) of field names.